app/services/idv/steps/verify_base_step.rb

```diff
@@ -24,16 +24,22 @@ def add_proofing_components
         )
       end
 
-      def check_ssn(pii_from_doc)
+      def check_ssn
+        pii_from_doc = flow_session[:pii_from_doc]
         result = Idv::SsnForm.new(current_user).submit(ssn: pii_from_doc[:ssn])
-        save_legacy_state(pii_from_doc) if result.success?
+
+        if result.success?
+          save_legacy_state(pii_from_doc)
+          flow_session.delete(:pii_from_doc)
```

Comment:
after the SSN verify step, then we go to the phone step, so we still need the PII in the session for that, right?

Reply:
It's hard to follow, but `save_legacy_state` copies the PII bundle in the session under a different key if the verify step is successful:

```ruby
        idv_session['applicant'] = pii_from_doc
```

My understanding is the phone step and later are looking in `'applicant'`, and `flow_session[:pii_from_doc]` is never read again after it is copied.

Reply:
My hope is to continue to pare down so we can have some more confidence and understanding in how/where we store PII in the session